From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 24 Jun 2016 16:51:41 +0200
Subject: [PATCH] Cleanup: Use dget instead of wget+dpkg-source.

We already install devscripts, which provides dget.

See [1] for using `dget -x -u` to accelerate/simplify code dev.

FIXME: Remove `-u` when we add Ubuntu signing keys to builder
docker keyring.

[1] https://debian-administration.org/article/
    504/Downloading_Debian_source_packages_easily

Change-Id: Ifbcac2a14c7b34216894adb7c5c4e921bf556e33
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 ovs_build/build-ovs-nsh-dpdk.sh | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/ovs_build/build-ovs-nsh-dpdk.sh b/ovs_build/build-ovs-nsh-dpdk.sh
index 1b42df1..3d7edea 100755
--- a/ovs_build/build-ovs-nsh-dpdk.sh
+++ b/ovs_build/build-ovs-nsh-dpdk.sh
@@ -17,10 +17,7 @@ sudo apt-get -y install devscripts dpkg-dev git wget
 rm -rf ${BUILD_HOME}; mkdir -p ${BUILD_HOME}
 
 cd ${BUILD_HOME}
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.dsc
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0.orig.tar.gz
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.debian.tar.xz
-dpkg-source -x dpdk_2.2.0-0ubuntu8.dsc
+dget -x -u https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.dsc
 
 # copy from debian/control
 sudo apt-get install -y debhelper \
@@ -49,10 +46,7 @@ cd ${BUILD_HOME}; sudo dpkg -i *.deb
 apt-get download libxenstore3.0
 
 cd ${BUILD_HOME}
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0.orig.tar.gz
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.dsc
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.debian.tar.xz
-dpkg-source -x openvswitch-dpdk_2.4.0-0ubuntu1.dsc
+dget -x -u https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.dsc
 
 # copy from debian/control
 sudo apt-get install -y autoconf \
